config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
Interestingly I've just been reading through https://github.com/exercism/v2-feedback/issues/36
and I'm going to update the config.json to include more side exercises.